modules/tinymce/src/plugins/template/test/ts/browser/DatesTest.ts

```diff
@@ -1,99 +1,95 @@
-import { Log, Logger, Mouse, Pipeline, Step, UiFinder, Waiter } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
-import { SugarElement } from '@ephox/sugar';
-import TemplatePlugin from 'tinymce/plugins/template/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/mcagar';
 
-UnitTest.asynctest('browser.tinymce.plugins.template.DatesTest', (success, failure) => {
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/template/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-  TemplatePlugin();
-  SilverTheme();
+import { insertTemplate } from '../module/InsertTemplate';
+import { Settings } from '../module/Settings';
 
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
+describe('browser.tinymce.plugins.template.DatesTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
```

Comment:
```suggestion
  const hook = TinyHooks.bddSetupLight<Editor>({
```

This is to keep it consistent with the previous version.